webapp/channels/src/components/channel_notifications_modal/channel_notifications_modal.tsx

```diff
@@ -36,508 +63,301 @@ type Props = PropsFromRedux & {
-                                        onSubmit={this.handleSubmitChannelAutoFollowThreads}
-                                        onUpdateSection={this.updateSection}
-                                        serverError={serverError}
-                                        isGM={isGM}
-                                    />
-                                </>
-                                }
-                                <div className='divider-dark'/>
-                            </div>
-                        </div>
-                    </div>
-                    {serverErrorTag}
-                </Modal.Body>
-            </Modal>
+                <RefreshIcon
+                    size={14}
+                    color={'currentColor'}
+                />
+                {formatMessage({
+                    id: 'channel_notifications.resetToDefault',
+                    defaultMessage: 'Reset to default',
+                })}
+            </button>
         );
-    }
+    }, [props.currentUser, settings]);
+
+    const settingsAndAlertBanner = settings.mark_unread === 'all' ? (
+        <>
+            <div className='channel-notifications-settings-modal__divider'/>
+            <SectionCreator
+                title={DesktopNotificationsSectionTitle}
+                description={DesktopNotificationsSectionDesc}
+                content={DesktopNotificationsSectionContent}
+                titleSuffix={resetToDefaultBtn('desktop')}
+            />
+            <div className='channel-notifications-settings-modal__divider'/>
+            <SectionCreator
+                title={MobileNotificationsSectionTitle}
+                description={MobileNotificationsSectionDesc}
+                content={MobileNotificationsSectionContent}
+                titleSuffix={resetToDefaultBtn('push')}
+            />
+        </>
+    ) : (
+        <AlertBanner
+            mode='info'
+            variant='app'
+            customIcon={
+                <BellOffOutlineIcon
+                    size={24}
+                    color={'currentColor'}
+                />
+            }
+            title={
+                <FormattedMessage
+                    id='channel_notifications.alertBanner.title'
+                    defaultMessage='This channel is muted'
+                />
+            }
+            message={
+                <FormattedMessage
+                    id='channel_notifications.alertBanner.description'
+                    defaultMessage='All other notification preferences for this channel are disabled'
+                />
+            }
+        />
+    );
+
+    return (
+        <Modal
+            dialogClassName='a11y__modal channel-notifications-settings-modal'
+            show={show}
+            onHide={handleHide}
+            onExited={handleExit}
+            role='dialog'
+            aria-labelledby='channelNotificationModalLabel'
+            style={{display: 'flex', placeItems: 'center'}}
```

Comment:
Could you move these to the CSS, if possible?

Reply:
Would you mind taking a look at this one still? Everything else I cared about from my previous review was resolved otherwise, but I'd still like to avoid inline styling

Reply:
Yeah, it's not possible to have added this to override the default inline style getting applied from Bootstrap

Reply:
I mean only other option to use is important which is even worse in my opinion

Reply:
That makes sense. I think I might prefer `!important` here just to keep all the styling in the CSS, but I'm 0/5 about that